Update/omake_02.txt

```diff
@@ -1726,7 +1726,7 @@ void main()
 	PlaySE(4, "S20/09/440900118", 128, 64);
 	OutputLine(NULL, "…『鬼隠し』でも、今回の『綿流し』でも。",
 		   NULL, " ...Both last time in Onikakushi, and this time in Watanagashi.", Line_WaitForInput);
-	PlaySE(4, "S02/00/440900118_2", 128, 64);
+	PlaySE(4, "S02/00/440900119", 128, 64);
```

Comment:
Yeah, apparently @Norgus made a small mistake here. He should have updated the PlaySE line above instead of below. https://github.com/higurashi-mod/watanagashi/commit/ffd3319b8af9ed43fbc59cfe89cd85d853aee42a#diff-4844dd6b98365311b3f792d132de46dcL1723